.travis.yml

```diff
@@ -10,11 +10,13 @@ services:
   - docker
 before_script: 
   - pip install docker-compose
+  - pip install -r server/requirements.txt
 jobs:
   include:
     - stage: "Build docker image"
       if: branch = develop
       script: 
+        - python -m unittest server/unit_test/test_cases.py
```

Comment:
We should create another stage, before building docker image, where the tests are triggered, as the tests are not part of the building process.